packages/react-signature-modal/package.json

```diff
@@ -32,15 +32,32 @@
     "modal"
   ],
   "devDependencies": {
-    "prop-types": "^15.6.0",
+    "@babel/core": "^7.17.8",
+    "@babel/plugin-proposal-class-properties": "^7.16.7",
+    "@babel/preset-env": "^7.16.11",
+    "@babel/preset-react": "^7.16.7",
+    "@babel/register": "^7.17.7",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.7",
+    "auto-changelog": "^2.4.0",
+    "babel-loader": "^8.2.4",
+    "chai": "^4.3.6",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-react": "^7.29.4",
+    "eslint-plugin-react-hooks": "^4.4.0",
+    "jsdom": "^18.0.0",
+    "mocha": "^9.2.2",
+    "prop-types": "^15.8.1",
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
-    "react-modal": "^3.11.2"
+    "react-modal": "^3.14.4",
+    "webpack": "^5.70.0",
+    "webpack-cli": "^4.9.2"
   },
   "peerDependencies": {
     "prop-types": "^15.6.0",
```

Comment:
No new majors here or in react-modal, eh?

Reply:
Yeah no new majors there. I used `yarn upgrade-interactive --latest` and chose the ones that don't have such a big impact and that's what we got.